evaluate/report/csv.go

```diff
@@ -46,16 +46,17 @@ func GenerateCSV(formatter CSVFormatter) (csvData string, err error) {
 
 // Header returns the header description as a CSV row.
 func (a *AssessmentStore) Header() (header []string) {
-	return append([]string{"model", "language", "repository", "task", "score"}, metrics.AllAssessmentKeysStrings...)
+	return append([]string{"model", "language", "repository", "task", "cost", "score"}, metrics.AllAssessmentKeysStrings...)
```

Comment:
nit: since the cost is part of the model I would put it up front to where the model is